packages/editor/src/plugins/text/components/text-leaf-with-placeholder.tsx

```diff
@@ -30,6 +30,7 @@ export function TextLeafWithPlaceholder(
       <span
         className="pointer-events-none absolute -mt-1 block w-full text-gray-400 [user-select:none]"
         contentEditable={false}
+        style={{ userSelect: 'none' }}
```

Comment:
```suggestion
        className="pointer-events-none absolute -mt-1 block w-full text-gray-400 select-none"
        contentEditable={false}
```

This should work, did it not?